assets/js/common/donations.js

```diff
@@ -133,6 +139,21 @@ if (typeof Mozilla === 'undefined') {
         }
     };
 
+    /**
+     * Checks and applies any position parameter to the window's scrollY
+     * This makes the silly page refresh on the donation modal look less jarring
+     */
+    Donation.CheckForPosition = function () {
```

Comment:
Hmm... I'm not sure this is actually working? In my testing on Firefox, it didn't seem to change the behaviour. I still get click download beta -> page reload showing top of page with modal -> after closing modal, page resets to bottom.